biz/services/udp_server/server.go

```
@@ -0,0 +1,106 @@
+type connection struct {
+	id         uint64
+	remoteAddr net.Addr
+	time       time.Time
+}
+
+type UDPServer struct {
+	gnet.BuiltinEventEngine
+
+	eng            gnet.Engine
+	id             *uid64.Generator
+	stop           chan struct{}
```

golangci-lint (1.62.2)

field `stop` is unused

(unused)